CMakeLists.txt

```diff
@@ -48,22 +48,29 @@ SET (CMAKE_VERBOSE_MAKEFILE 0) # Use 1 for debugging, 0 for release
 #
 # Project Output Paths
 #
-#MESSAGE(${PROJECT_SOURCE_DIR})
-#SET (MAINFOLDER ${PROJECT_SOURCE_DIR})
 SET (LEMONADE_DIR ${PROJECT_SOURCE_DIR})
-#MESSAGE(${LEMONADE_DIR})
-SET (EXECUTABLE_OUTPUT_PATH "${LEMONADE_DIR}/build/bin")
-SET (LIBRARY_OUTPUT_PATH "${LEMONADE_DIR}/lib")
+
+#
+# Check if LEMONADE_DIR is given
+#
+if (DEFINED INSTALLDIR_LEMONADE)
+    message("INSTALLDIR_LEMONADE set to "${INSTALLDIR_LEMONADE})
+else()
+    SET (INSTALLDIR_LEMONADE "${LEMONADE_DIR}/build/")
+    message("INSTALLDIR_LEMONADE set automatically to "${INSTALLDIR_LEMONADE}". Use -DINSTALLDIR_LEMONADE=/path/to/install/LeMonADE/ for other destination")
+endif()
+
+SET (EXECUTABLE_OUTPUT_PATH "${INSTALLDIR_LEMONADE}/bin")
+SET (LIBRARY_OUTPUT_PATH "${INSTALLDIR_LEMONADE}/lib")
```

Comment:
We can also provide symbolic links of the library for highlighting different version, e.g. libLeMonADE.a -> symbolic links latest release -> libLeMonADE-1.0.0.a This approach is used by the LeMonADE-Viewer-project. Should I do this?